### ISSUE TITLE
Be --quiet option for auter

### ISSUE THREAD

**Issue by exploide** [rackerlabs/auter#141]
I have auter in a cron job, including auto reboot. Every time it installs updates, I receive an email like

> Shutdown scheduled for Mon 2018-04-16 05:00:11 CEST, use 'shutdown -c' to cancel.

Silencing this via `> /dev/null` stdout redirection does not work. Silencing this via `2> /dev/null` stderr redirection works but would also suppress real errors I would like to see. Maybe auter could have a `--quiet` flag or configuration option and silences `shutdown` when given.

**Comment by reaperzn** [comment]
Hi @exploide thanks for raising this. I have not looked into this too much but there should be a --no-wall option for the reboot/shutdown command. This will mean that we do not need to redirect stderr to /dev/null which may result in missing legitimate errors. 

I will look into this when I get some time but maybe @nickrhodes-gh or @popindavibe  could whip up a quick PR to add a --quiet option which would adjust this line:
https://github.com/rackerlabs/auter/blob/405f54359800c1f002f9d7f019f2b12ac74f3c06/auter#L150

**Comment by reaperzn** [comment]
Hi @exploide 

So it turns out this is a bit more tricky that I initially thought it would be. For RHEL7/CentOS7, Debian and Ubuntu this is easy enough using the ```--no-wall``` option for the shutdown command. For RHEL6/CentOS6 this option is not available. I have added the --no-wall option in PR #147 

HOWEVER.... This actually seems be cron which is emailing you the stdout from the shutdown command too. It is probably worth adding the following line to the top of your auter cron file:
```MAILTO=""```

Please do let us know if this resolves your issue.

Thanks

**Comment by exploide** [comment]
Hi, thanks for your effort!

That the `--no-wall` option is only available on RHEL7 is no issue for me since I typically don't care about legacy systems :grin:

But did you test if this really has an impact? I don't know if it simply does not work or I misunderstood the desired behavior, but when I call `shutdown --no-wall -r 5` on my Fedora machine, the same output appears as when invoked w/o `--no-wall`. However, I did not test what would happen when this lands in auter and is invoked from there.

I don't fully understand your second point, of course it is cron sending the emails. But wouldn't `MAILTO=""` imply that it won't send email in any case? Even if an error occurs? Then I can also stay with my current workaround (stderr redirection to /dev/null).

If you see no good way to improve the behavior, you may also just close this issue. I only wanted to ask if someone has an idea.

**Comment by reaperzn** [comment]
Hi @exploide 

So there are 2 separate points here. There is the wall message that will be sent to all active sessions. This will be suppressed with the ```--no-wall``` option. 

The second part which you are getting via cron is the actual output from the shutdown command which is being sent to STDOUT. So as an example:
This is the output of the command as it is now. (Note I used -k to only print messages)
```# /sbin/shutdown -k +2 "auter: System reboot to apply updates"
Shutdown scheduled for Wed 2018-05-02 10:41:25 UTC, use 'shutdown -c' to cancel.

Broadcast message from root@paolo-auter-test7 (Wed 2018-05-02 10:39:25 UTC):

auter: System reboot to apply updates
The system is going down for power-off at Wed 2018-05-02 10:41:25 UTC!
```

This is with the --no-wall option:
```# /sbin/shutdown --no-wall -k +2 "auter: System reboot to apply updates"
Shutdown scheduled for Wed 2018-05-02 10:42:14 UTC, use 'shutdown -c' to cancel.```

What I have done in the latest commit in the PR is to redirect the output of the shutdown command only. So any other errors from auter will not be suppressed. 

You are correct, changing the MAILTO option will still hide all other errors so not an option. Your current workaround, are you redirecting stderr from the auter command or did you modify the code to only redirect from the shutdown command?

**Comment by exploide** [comment]
> What I have done in the latest commit in the PR is to redirect the output of the shutdown command only. So any other errors from auter will not be suppressed. 

This sounds good :+1:

> Your current workaround, are you redirecting stderr from the auter command or did you modify the code to only redirect from the shutdown command?

I redirect stderr from the auter command.

**Comment by rhodesn** [comment]
Resolved in #147.
